actor-server/actor-fs-adapters/src/main/scala/im/actor/server/file/s3/S3StorageAdapterConfig.scala

```diff
@@ -13,7 +13,8 @@ object S3StorageAdapterConfig {
       bucketName ← config.get[Try[String]]("default-bucket")
       key ← config.get[Try[String]]("access-key")
       secret ← config.get[Try[String]]("secret-key")
-    } yield S3StorageAdapterConfig(bucketName, key, secret)
+      endpoint ← config.get[Try[String]]("endpoint")
```

Comment:
this will always fail if no default value set. Please put it into reference.conf.